README

```diff
@@ -208,6 +208,11 @@ Default:	NULL
 Description:	The URL to use when performing a proxy validation.  This is currently
 		an unimplemented feature, so setting this will have no effect.
 
+Directive:	CASDisableRedirectAfterValidation
+Default:	On
+Description:	Whether to redirect to the same URL after ticket validation, but without the 
+		ticket in the parameter.  Defaults to on
```

Comment:
The default should be Off here, right?

The description seems to be from the java cas client, which I don't find terribly descriptive. I'd like to see something about what we do by default and what this directive does to change that behavior.